slices/libpam-modules.yaml

```diff
@@ -0,0 +1,84 @@
+package: libpam-modules
+
+essential:
+  - libpam-modules_copyright
+
+slices:
+  config:
+    contents:
+      /etc/security/access.conf:
+      /etc/security/faillock.conf:
+      /etc/security/group.conf:
+      /etc/security/limits.conf:
+      /etc/security/limits.d/:
+      /etc/security/namespace.conf:
+      /etc/security/namespace.d/:
+      /etc/security/namespace.init:
+      /etc/security/pam_env.conf:
+      /etc/security/sepermit.conf:
+      /etc/security/time.conf:
+
+  libs:
+    essential:
+      - libaudit1_libs
```

Comment:
Hmmm.... some of the `.so` here does mention the binaries in `libpam-modules-bin`. For example:

```
root@acdf7f8ae575:~/libpam-modules# strings ./usr/lib/x86_64-linux-gnu/security/pam_extrausers.so | grep pam_extrausers_chkpwd
/usr/sbin/pam_extrausers_chkpwd
pam_extrausers_chkpwd waitpid returned %d: %m
pam_extrausers_chkpwd abnormal exit: %d
read pam_extrausers_chkpwd output error %d: %m
```

I kept it identical to 24.04 because the files have not changed since then. This was added in #142 by @Meulengracht. If we need to change it, I think we will need to do so in 24.04 first. I would like to keep that out of the scope of forward-porting. But something we should definitely check out.